law.cfg

```diff
@@ -123,5 +123,6 @@ worker-disconnect-delay: 30
 ping_interval: 20
 wait_interval: 20
 check_unfulfilled_deps: False
+cache_task_completion: True
```

Comment:
Ah, this got in there by accident, but I think it could stay :) I added a completion cache to luigi last week (https://github.com/spotify/luigi/pull/3178) which could help us speed up things. It will be part of the next release.

Reply:
Ah ok, cool! Good to know